package.json

```diff
@@ -27,7 +27,8 @@
     "standard": "standard",
     "test": "run-p standard karma wdio",
     "version": "npm run build && git add -A dist",
-    "wdio": "cross-env NODE_ENV=test wdio test/wdio.config.js && git checkout dist/"
+    "wdio:test": "cross-env NODE_ENV=test wdio test/wdio.config.js",
+    "wdio": "run-s build wdio:test && git checkout dist/"
```

Comment:
Is there a reason to use `run-s` here over `npm run build wdio:test && git checkout dist`?

Reply:
Ah, I didn't realise that `npm run` could run multiple commands itself. I will update this line.

Reply:
Argh, I think I might be talking rubbish. My brain did not parse that correctly, I was reading it as `wdio:test` being an argument to `npm build` rather than `build` and `wdio:test` being two different tasks 🤦